docs/user-guide/cli-using-team-managing-credential-security.md

```diff
@@ -1,15 +1,77 @@
 # Managing credential security
 
-When you first run the `zowe config init --global-config` command, the `profiles.base.properties.user` and `profiles.base.properties.password` fields are defined to the "secure" array in your configuration file, which helps to ensure that the username and password are stored securely on your computer.
+## Secure credential storage
 
-To store or update values for the secure fields (for example, when you want to change your username and password), issue the `zowe config secure` command. If, for example, you want to update several property values in a long list of properties, press Enter to skip a field.
+With the introduction of team profiles in Zowe CLI V2, the **Secure Credential Store (SCS) Plug-in** is deprecated. The `zowe scs` command group is obsolete.
 
-To secure a specific field, issue `zowe config set --secure <property-path>`. For example, `zowe config set --secure profiles.base.properties.password`. When you issue the command for an option that is already secured, the CLI prompts you to enter a new option value.
+Secure credential encryption is now included with the Zowe CLI core application. When a command using a profile with missing `user` and `password` information is issued, Zowe CLI V2 prompts you to enter the username and password by default. To save secure properties and values in the Secure Credential Store, use the `zowe config` command group.
```

Comment:
Suggest change:
"... Zowe CLI V2 prompts you to enter the username and password, which are then stored securely by default."

Reply:
> To save secure properties and values in the Secure Credential Store, use the `zowe config` command group.

Should we avoid using the terms `Secure Credential Store` and instead use something (ideally OS agnostic) like Vault (windows) or  Keychain (Mac)?

And should we point people to the zowe config secure section below?

Reply:
@zFernand0 -- as discussed w/ the team, we have agreed to define "secure credential store" in the [Zowe glossary](https://docs.zowe.org/stable/appendix/zowe-glossary) as the separate component that it is, explaining that it is no longer a separate plug-in but part of core Zowe V2 CLI.